components/SuperAdminPage/ProfileInfo.vue

```diff
@@ -0,0 +1,18 @@
+<script setup>
+        const {props} = defineProps({
+        username: String,
+        role: String,
+    })
+</script>
+
+<template>
+    <div class="w-2/3">
```

Comment:
```suggestion
    <section class="w-2/3">
```
Semantic HTML over just `div`s is better for accessibility.